app.js

```diff
@@ -31,46 +32,38 @@ const config = require("config");
 const session = require("express-session");
 const redis = require("redis");
 
-
-
 const app = express();
 
 app.use(logger("dev"));
-app.use(express.raw({type: 'application/octet-stream', limit: "2gb"}));
-app.use(express.json({limit: "512kb"}));
+app.use(express.raw({ type: "application/octet-stream", limit: "2gb" }));
+app.use(express.json({ limit: "512kb" }));
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use("/demo", express.static("public"));
 app.use("/static/img/logos", express.static("public/img/logos"));
 
 // Security defaults with helmet
 app.use(helmet());
-if(config.get("server.useRedis")){
+if (config.get("server.useRedis")) {
   // Messy but avoids any open file handles.
   const redisClient =
-  process.env.NODE_ENV !== "test"
-    ? redis.createClient({
-        password: config.get(process.env.NODE_ENV + ".redisPass"),
-      })
-    : require("redis-mock").createClient();
+    process.env.NODE_ENV !== "test"
+      ? redis.createClient({
+          url: config.get(process.env.NODE_ENV + ".redisUrl"),
```

Comment:
A password should still absolutely be required for redis, not just the URL.

Reply:
The password is included in the url, the url is in the form of redis://<user>:<password>@<service-url>:<port> eg. redis://root:testing@redis:6379